ui/components/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@perses-dev/components",
```

Comment:
I imagine I'm going to be spending **_a lot_** of time in this package :smile:
I can take care of moving some of the chart components over if you'd like?

Reply:
Yeah, that would be great. Once we get this PR merged, you can definitely do that. 👍

Reply:
yup I opened a discussion here : https://github.com/perses/perses/discussions/331. I tried to find a clean solution and came up with different option.
It will be cool to have a consensus around this topic before starting any work around the CI/CD (which shouldn't be a big effort, since it just misses the release job(s))